.gitignore

```diff
@@ -8,3 +8,7 @@ nosetests.xml
 eggs/
 downloads/
 .idea
+.venv
```

Comment:
Die `.venv` krijg je volgens mij alleen bij een specifieke setting. Anders wordt het ergens in je homedir gegroepeerd neergezet (`~/.local/venvs/*` als ik het goed heb). Een `.venv` lokaal vind ik zelf een goed idee, maar even kijken of dat niet een globale setting bij jou is die je ergens in dit project ook aan moet geven.

**Update**: ik zie verderop dat er binnenkort `--local` komt. En dat het in de README gedocumenteerd is. Zit wel goed zo, dus.